src/navigation/MainTabNavigation.jsx

```diff
@@ -55,6 +60,12 @@ const tabs = {
   },
 };
 
+if (process.env.NODE_ENV === 'development') {
```

Comment:
you can add optional keys to an object with the following syntax:
```js
{
  ...(process.env.NODE_ENV === 'development' && { Playground: PlaygroundStack })
}
```

Reply:
Thanks! I've updated that. Please review again.